plumber_rest_api/api/main.R

```diff
@@ -37,7 +37,7 @@ function(req, res){
 #* @param id:string
 #' @post /user_log
 #' @serializer json
-function(token, id, data, res){
+function(token = NULL, id = NULL, data, res){
```

Comment:
Since we rearranged arguments in line 53, I think it will be more consistent if we also put data first here.

Reply:
I removed from all, it's not following openAPI standard and it seems it's no longer a problem with tests!